src/features/canvass/components/CanvassSelectAreaPage.tsx

```diff
@@ -0,0 +1,144 @@
+'use client';
+
+import { FC, useState } from 'react';
+import React from 'react';
+import { ArrowForwardIos } from '@mui/icons-material';
+import {
+  Avatar,
+  Box,
+  CircularProgress,
+  Divider,
+  List,
+  ListItem,
+  ListItemButton,
+  Typography,
+} from '@mui/material';
+import { useRouter } from 'next/navigation';
+
+import useMyCanvassAssignments from '../hooks/useMyAreaAssignments';
+import { ZetkinAreaAssignment } from '../../areaAssignments/types';
+import useOrganization from 'features/organizations/hooks/useOrganization';
+import ZUIFutures from 'zui/ZUIFutures';
+import oldTheme from 'theme';
+import { Msg } from 'core/i18n';
+import messageIds from '../l10n/messageIds';
+import useAssignmentAreas from 'features/areaAssignments/hooks/useAssignmentAreas';
+
+const Page: FC<{
+  assignment: ZetkinAreaAssignment;
+}> = ({ assignment }) => {
+  const orgFuture = useOrganization(assignment.organization_id);
+  const router = useRouter();
+  const areas = useAssignmentAreas(assignment.organization_id, assignment.id);
+  const [loadingAreaId, setLoadingAreaId] = useState<number | null>(null);
+  return (
+    <ZUIFutures futures={{ org: orgFuture }}>
+      {({ data: { org } }) => (
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            height: '100dvh',
+            width: '100vw',
+          }}
+        >
+          <Box
+            alignItems="center"
+            bgcolor={oldTheme.palette.background.paper}
+            display="flex"
+            justifyContent="space-between"
+            padding={2}
+          >
+            <Box>
+              <Typography variant="body1">{assignment.title}</Typography>
```

Comment:
I would like a back button here, to go back to the assignment instructions etc. I don't know if this is something you have already discussed and maybe there's a reason not to do it?

Reply:
We didn't think about it, but I agree it’s a good idea to implement it to ensure smooth navigation between pages when needed.